app/containers/ProfilePage/index.js

```diff
@@ -6,28 +6,40 @@
 
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { Map } from 'immutable';
+
+
 import PasswordChangeForm from '../../components/PasswordChangeForm';
 import { changePassword } from './actions';
+import { passwordChangePropType } from '../../components/PasswordChangeForm/constants';
+
 
 function ProfilePage(props) {
   return (
     <div>
       <PasswordChangeForm
         onSubmit={(oldPassword, password) => props.dispatch(changePassword(oldPassword, password))}
```

Comment:
Did you considered changing saga to `@graphql` like we do in other places where we can? Then exporting props to const, whole machinery for storing and fetching data from store shouldn't be necessary.